src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx

```diff
@@ -37,41 +39,58 @@ function ConditionFields({
   onChange,
 }: Props) {
   return (
-    <React.Fragment>
+    <Wrapper>
       {conditions.map(({match, category}, index) => {
-        const displayDescription = index === 0;
         const showLegacyBrowsers = category === DynamicSamplingInnerName.LEGACY_BROWSERS;
         return (
-          <Fields key={index}>
-            <SelectField
-              label={displayDescription ? t('Category') : undefined}
-              help={displayDescription ? t('This is a description') : undefined}
-              name={`category-${index}`}
-              value={category}
-              onChange={value => onChange(index, 'category', value)}
-              choices={categoryOptions}
-              inline={false}
-              hideControlState
-              showHelpInTooltip
-              required
-              stacked
-            />
-            <TextField
-              label={displayDescription ? t('Match Conditions') : undefined}
-              help={displayDescription ? t('This is a description') : undefined}
-              placeholder={
-                showLegacyBrowsers ? t('No match condition') : 'ex. 1* or [I3].[0-9].*'
-              }
-              name={`match-${index}`}
-              value={match}
-              onChange={value => onChange(index, 'match', value)}
-              disabled={showLegacyBrowsers}
-              inline={false}
-              hideControlState
-              showHelpInTooltip
-              required
-              stacked
-            />
+          <FieldsWrapper key={index}>
+            <Fields>
+              <SelectField
+                label={t('Category')}
+                help={t('This is a description')}
+                name={`category-${index}`}
+                value={category}
+                onChange={value => onChange(index, 'category', value)}
+                choices={categoryOptions}
+                inline={false}
+                hideControlState
+                showHelpInTooltip
+                required
+                stacked
+              />
+              <StyledField
+                label={t('Match Conditions')}
+                help={t('This is a description')}
```

Comment:
I would omit this so it doesn't get left behind by accident.

Reply:
@jan-auer please do you already have a description for these fields?